R/dataframe.R

```diff
@@ -38,10 +38,10 @@ lime.data.frame <- function(x, model, bin_continuous = TRUE, n_bins = 4, quantil
   explainer$bin_cuts <- setNames(lapply(seq_along(x), function(i) {
     if (explainer$feature_type[i] %in% c('numeric', 'integer')) {
       if (quantile_bins) {
-        bins <- quantile(x[[i]], seq(0, 1, length.out = n_bins + 1))
+        bins <- quantile(x[[i]], seq(0, 1, length.out = n_bins + 1),na.rm=T)
         bins[!duplicated(bins)]
       } else {
-        d_range <- range(x[[i]])
+        d_range <- range(x[[i]],na.rm=T)
```

Comment:
I have the same fix locally - just remove your modifications here and lets focus this PR on xgboost support :-)